install/install.sh

```diff
@@ -41,24 +45,26 @@ fi
 
 # Remove old extension
 spicetify config extensions bloom.js- -q
-if [[ -e "$spicePath/Extensions/bloom.js" ]]; then
+extensionPath="$spicePath/Extensions/bloom.js"
+if [[ -e "$extensionPath" || -h "$extensionPath" ]]; then
   rm "$spicePath/Extensions/bloom.js"
 fi
 
 # Download latest master
 zipUri="https://github.com/nimsandu/spicetify-bloom/archive/refs/heads/master.zip"
-zipSavePath="/tmp/bloom-main.zip"
+zipSavePath=`mktemp`
```

Comment:
You're using `mktemp` here but it's better if you add a check for it as well.

Reply:
> Do you mean to check that the directories don't exist?

No, something like this;
```bash
if [ -z "$(which mktemp)" ]; then
    # mktemp doesn't exist
    # b/126
    exit 5
fi
```

Reply:
Done, although it would be weird not to have it installed as it's included in the coreutils

Reply:
Now gonna merge this PR.